lib/Twig/NodeVisitor/Optimizer.php

```diff
@@ -45,6 +46,7 @@ public function __construct($optimizers = -1)
         }
 
         $this->optimizers = $optimizers;
+        $this->isPHP540RCorHigher = version_compare(phpversion(), '5.4.0RC1', '>=');
```

Comment:
Thank you for the feedback. I've made the adjustments. Let me know what you think :)